src/ServiceControl.Monitoring/Infrastructure/EndpointInputQueue.cs

```diff
@@ -2,7 +2,11 @@
 {
     using System;
 
+    // it's not entirely clear whether we can rename this type or not, given that it seems to be used for storage
+    // all references seem to indicate it should be called something like QueueId
+#pragma warning disable CA1711 // Identifiers should not have incorrect suffix
     public readonly struct EndpointInputQueue : IEquatable<EndpointInputQueue>
+#pragma warning restore CA1711 // Identifiers should not have incorrect suffix
```

Comment:
@andreasohlund are you OK with this? I've tried asking around about whether this type can be renamed, but no answer yet. I figure we can commit this for now.